src/pages/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import BookItem from '../components/BookItem';
```

Comment:
- [ ] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile: